internal/magic/text.go

```diff
@@ -120,6 +120,17 @@ var (
 	)
 	// Rtf matches a Rich Text Format file.
 	Rtf = prefix([]byte("{\\rtf1"))
+	// Vtt matches a Web Video Text Tracks (WebVTT) file. See
```

Comment:
I guess you wanted to add a link to IANA here. Feel free to do so.

Reply:
That's exactly what it was. Thank you for noticing.